src/Badge/Badge.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { StyledBadge } from './Badge.styles';
+
+export interface Props {
+  background?: string;
+  color?: string;
+  size?: 'standard'; // TODO: fill this out more one design system has caught-up
```

Comment:
I haven't handled this prop yet as need a conversation about design system to happen first.